### PR TITLE
fix(zh-tw): remove some unrendered macros

### PR DESCRIPTION
**Pull request by yin1999**
### Description

remove some unrendered macros

### Related issues and pull requests

Part of: #13499

### CODE REVIEW

File: files/zh-tw/web/css/css_box_model/mastering_margin_collapsing/index.md [hunk 1]
```diff
@@ -83,11 +83,11 @@ slug: Web/CSS/CSS_box_model/Mastering_margin_collapsing
 
 以上這些內容都是發生在 Block-Level 的元素，設定 floating 和 absolutely positioned 的元素完全不用擔心邊界重疊的計算。
 
-## 標準文件
+## 規範
 
 {{Specifications}}
```
Comment (jasonren0403) [inline files/zh-tw/web/css/css_box_model/mastering_margin_collapsing/index.md:88]: not rendered in en-US, not for sure how to handle it 👀

Comment (yin1999) [reply files/zh-tw/web/css/css_box_model/mastering_margin_collapsing/index.md:88]: see https://github.com/mdn/translated-content/issues/13499#issuecomment-1573643030, let's keep it until the upstream has fix it.

Comment (jasonren0403) [reply files/zh-tw/web/css/css_box_model/mastering_margin_collapsing/index.md:88]: found a pr here https://github.com/mdn/content/pull/27113, maybe we can wait for this

Comment (yin1999) [reply files/zh-tw/web/css/css_box_model/mastering_margin_collapsing/index.md]: Removed in mdn/content#27147

**Review by jasonren0403 — APPROVED** [review]

**Pull request merged**